pkgs/servers/x11/xorg/default.nix

```diff
@@ -1992,7 +1992,7 @@ lib.makeScope newScope (self: with self; {
   }) {};
 
   # THIS IS A GENERATED FILE.  DO NOT EDIT!
-  xdm = callPackage ({ stdenv, pkg-config, fetchurl, libX11, libXau, libXaw, libXdmcp, libXext, libXft, libXinerama, libXmu, libXpm, xorgproto, libXrender, libXt }: stdenv.mkDerivation {
+  xdm = callPackage ({ stdenv, pkg-config, fetchurl, libX11, libXau, libXaw, libXdmcp, libXext, libXft, libXinerama, libXmu, libXpm, libxcrypt, xorgproto, libXrender, libXt }: stdenv.mkDerivation {
```

Comment:
the changes to this file will be overriden once `./generate-expr-from-tarballs.pl tarballs.list` is run, moving this fix to overrides.nix in https://github.com/NixOS/nixpkgs/pull/199912